blaze-core/src/test/scala/org/http4s/blaze/util/EntityBodyWriterSpec.scala

```diff
@@ -100,92 +99,106 @@ class ProcessWriterSpec extends Http4sSpec {
     runNonChunkedTests(tail => new CachingChunkWriter(new StringWriter(), tail, Task.now(Headers())))
   }
 
-  "ChunkProcessWriter" should {
+  "ChunkEntityBodyWriter" should {
     def builder(tail: TailStage[ByteBuffer]) =
-      new ChunkProcessWriter(new StringWriter(), tail, Task.now(Headers()))
-
-    "Not be fooled by zero length chunks" in {
```

Comment:
I thought those were testing complexity in the old implementation that melted away with the deoptimization, but now that you mention it, it was probably guarding against writing a chunk of 0 length.  I expect it to work because fs2 elides empty chunks, but it doesn't hurt to check.